src/index.html

```diff
@@ -1,13 +1,368 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link
+      type="Image/x-icon"
+      href="images/favicon.png"
+      rel="icon"
+    >
+    <title>
+      MyBike
+    </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <script src="scripts/main.js" defer>
+
+    </script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header" >
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top top-actions">
+            <a href="/" class="logo">
```

Comment:
Checklist | point 3 - logo should reload our page for home page. When we have "/" in href - we get 404 site didn;t find